assets/js/script.js

```diff
@@ -0,0 +1,353 @@
+let songEl = document.getElementById('song')
+let clackEl = document.getElementById('clack')
+
+let timeLeft = 185;
+let answerBtnIndividual = document.getElementById('btn1', 'btn2', 'btn3', 'btn4')
+
+
+//Saving variables
+let scoreName = document.getElementById('nameInput').value;
+let orderedListEL = document.getElementById('highScoreList');
+// let scoreGroup = JSON.parse(localStorage.getItem('highScores'))||[]
+let scoreGroup=[]
+// let orderedListEL = JSON.parse(localStorage.getItem(orderedListEL));
+
+
+
+
+
+
+//Start Game button function
+startButton.addEventListener("click", startGame)
+
+function startGame(){
+  startButton.classList.add("hide")
+  questionContainerEl.classList.remove("hide")
+  displayQuestion(currentQuestion)
+  songEl.volume = 0.6;
+  songEl.play()
+  corkEl.play()
+  let downloadTimer = setInterval(function(){
+    if(timeLeft <= 0){
+      clearInterval(downloadTimer);
+      document.getElementById("countdown").innerHTML = "Finished!";
+      endGame();
+    } else {
+      document.getElementById("countdown").innerHTML = timeLeft + " seconds remaining";
+    }
+    timeLeft -= 1;
+    }, 1000);
+}
+
+
+//Function for choosing an answer
+let chosenAnswer = document.getElementById("btn1","btn2","btn3","btn4")
+console.log(chosenAnswer)
+
+function chooseAnswer(event){
+  console.log(event.target)
+  let correctAnswer = questions[currentQuestion].correctAnswer
+  if(event.target.textContent == correctAnswer) {
+    console.log("correct!")
+    score+=50
+    currentScoreEl.textContent=score
+    //play ding
+    dingEl.play()
+  }
+  else {
+    console.log("wrong!")
+    corkEl.play()
+  }
+  currentQuestion++
+    if(currentQuestion >= questions.length) {
+      endGame();
+    }
+    else {
+      displayQuestion(currentQuestion)
+    }
+}
+
+//Show questions
+function displayQuestion(i){
+  document.getElementById("question").textContent = questions[i].question
+  document.getElementById("btn1").textContent = questions[i].answers[0]
+  document.getElementById("btn2").textContent = questions[i].answers[1]
+  document.getElementById("btn3").textContent = questions[i].answers[2]
+  document.getElementById("btn4").textContent = questions[i].answers[3]
+}
+
+
+
+//View High Scores function
+viewScoresButton.addEventListener("click", viewScores)
+
+function viewScores(){
+  endGame()
+}
+
+
+//End Game Function
+function endGame() {
+  document.querySelector(".start").style.display="none"
+  document.querySelector('#questionCard').classList.add('hide')
+  questionContainerEl.classList.add('hide')
+  console.log(questionContainerEl)
+  firstContainerEl.classList.add('hide')
+  endGameEl.classList.remove('hide')
+  timeLeft = 0;
+  endScoreEl.textContent=score
+  console.log("GAME OVER")
+}
+
+//Play again function
+
+playAgainBtnEl.addEventListener('click', playAgain)
+
+function playAgain () {
+  console.log(questionContainerEl)
+  // questionContainerEl.classList.remove("hide")
```

Comment:
Remove _**all instances.**_ of unused comments. Unused comments add unnecessary overheard. 

Old code should not be kept in comments, it should be tracked in Git.